src/reanimated2/commonTypes.ts

```diff
@@ -8,13 +9,22 @@ export interface StyleProps extends ViewStyle, TextStyle {
   [key: string]: any;
 }
 
+interface SharedValueInternals<T> {
```

Comment:
is it a common practice to prefix such types with `Internals`?

Reply:
I've tried looking it up in Google, but haven't found anything. I've just went here with a gut-feeling